proposals/wac-ucr/index.bs

```diff
@@ -962,6 +1357,12 @@ over 18. Her client's credential manager should see this and select the
 credential revealing the least amount of information needed to access the
 resource.
 
+<div class="assertion">
+
+* [[#req-vc]]
```

Comment:
#req-vc" is "§ 3.2.1 The system shall allow access to be limited to an agent based on the agent’s possession of a certain verifiable credential or capability."
But that is not the main requirement here. The main requirement is that "The system shall allow the client to read the access control rules in order to be able to make a decision on what credentials to use if any."
This probably needs a section by itself on "Access to access control rules"

Reply:
Resolved in https://github.com/solid/authorization-panel/commit/f3a636936623033bad279d3cea53dc35c2d398cc, per [discussion](https://github.com/solid/authorization-panel/pull/167/files?short_path=e66d5b0#diff-e66d5b08e777fcb77d751f43b35a40f779e28776d19fde98601f334c1105e56a)